build/lib/release.sh

```diff
@@ -296,6 +303,26 @@ function kube::release::build_hyperkube_image() {
   fi
 }
 
+function kube::release::build_conformance_image() {
+  local -r arch="$1"
+  local -r registry="$2"
+  local -r version="$3"
+  local -r save_dir="${4-}"
+  kube::log::status "Building conformance image for arch: ${arch}"
+  ARCH="${arch}" REGISTRY="${registry}" VERSION="${version}" \
+    make -C cluster/images/conformance/ build >/dev/null
+
+  local conformance_tag="${registry}/conformance-${arch}:${version}"
+  if [[ -n "${save_dir}" ]]; then
+    "${DOCKER[@]}" save "${conformance_tag}" > "${save_dir}/conformance-${arch}.tar"
+  fi
+  if [[ -z "${KUBE_DOCKER_IMAGE_TAG-}" || -z "${KUBE_DOCKER_REGISTRY-}" ]]; then
+    # not a release
+    kube::log::status "Deleting conformance image ${conformance_tag}"
+    "${DOCKER[@]}" rmi "${conformance_tag}" &>/dev/null || true
+  fi
+}
+
 # This will take binaries that run on master and creates Docker images
```

Comment:
This comment is now stale.

Reply:
unrelated right? let's tackle this in another PR

Reply:
it's only stale now? /shrug